CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+# 27.0.0 [#826](https://github.com/openfisca/openfisca-core/pull/826)
+
+#### Breaking changes
+
+- No longer honor calls to set_input for variables which have an end date and for periods after that date.
+
+#### Migration guide
+
+This change is not expected to actually break any reusing code, as it seems inconsistent to specify an end date for a variable and use it in computations for periods past that date. However, nothing has previously ruled out relying on that behaviour.
```

Comment:
Meh about the "it seems inconsistent to...". This is quite judgmental, and I was a user who ended up needing to migrate code, this sentence would sound bitter to me.  
And, to be a little sassy, I could reply to this sentence by "It seems a little inconsistent to explicitly specify inputs for  a variable, and to want these inputs to be ignored", which is the goal of this PR 😛

Reply:
@fpagnoux : when the end.date change comes from a reform and you want to use the same initialization data, this is clearly what you end doing !

Reply:
@fpagnoux I agree with you… even though I wrote those words. I'll rephrase to be more congruent with that inner conflict. "I am large, I contain multitudes."